docs/posts/ibis-v8-release/index.qmd

```diff
@@ -0,0 +1,269 @@
+---
+title: "Ibis v8: streaming and more!"
+author: "Ibis team"
+date: "2024-02-05"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis v8 marks the first release of data streaming backends in Ibis! We're
+excited to begin bridging the gap between batch and streaming systems, starting
+with [Apache Flink](https://flink.apache.org/) and
```

Comment:
maybe specifically call out Flink as a streaming backend and RisingWave as a streaming database backend.